### PR TITLE
Adds Jarate, the Jar-Based Karate (And weaponized urine)

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

Adds a foam chemical grenade full of a new reagent you may have heard of called urine. It adds an EXTREMELY negative moodlet (-20 mood) if you come into contact with it.

## Why It's Good For The Game

Weaponized mood debuffs is actually an interesting concept and also reference and also funny meme. Currently not attainable in game normally.

Jarate? NOOOOOOOOOOOOOOOOO!!

## Changelog
:cl: Tupinambis
add: Adds Jarate, the jar based karate
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
Unfinished, will consider finishing it if this doesn't get immediately closed as a meme PR.

I'd like to add this to the traitor uplink.

File: code/game/objects/items/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -580,6 +580,23 @@
 	beakers += B1
 	beakers += B2
 
+/obj/item/grenade/chem_grenade/jarate
+	name = "jarate"
+	desc = "Jar-Based Karate."
+	stage = READY
+
+/obj/item/grenade/chem_grenade/holy/Initialize()
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/grenades/chem_grenade.dm]: Holy?

File: code/datums/mood_events/generic_negative_events.dm [hunk 2]
```diff
@@ -247,6 +247,11 @@
 	mood_change = -6
 	timeout = 1200
 
+/datum/mood_event/jarate
+	description = "<span class='boldwarning'>JARATE? NOOOOOOOOOOOOOOOOOO!!!.</span>\n"
```
Comment (ghost) [inline code/datums/mood_events/generic_negative_events.dm]: .

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -2116,6 +2116,19 @@
 	color = "#BCC740" //RGB: 188, 199, 64
 	taste_description = "plant dust"
 
+/datum/reagent/urine
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:2119]: this is very very unsettling

Comment (Tupinambis) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2119]: urine trouble

Comment (lolman360) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2119]: @izzyinbox

**Comment by silicons** [comment]
as funny as this is this is a no from me because why the fuck would we add urine as a reagent?

**Comment by davidchan13** [comment]
ITT Tupinambis tries to sneak in a piss fetish.

**Comment by Ghommie** [comment]
Can't merge such a slippery slope, might end up jarate'd. :joy:

**Pull request closed without merging**